src/components/bcf-topic-form/bcf-topic-snapshots/BcfTopicSnapshots.vue

```diff
@@ -18,11 +18,25 @@
 
     <template v-else>
       <div class="bcf-topic-snapshots__create">
-        <BcfTopicSnapshotsActions
-          :viewpoints="viewpoints"
-          :getViewers="getViewers"
-          @create-viewpoint="$emit('create-viewpoint', $event)"
-        />
+        <BIMDataButton color="primary" fill radius @click="createViewpoints">
+          <BIMDataIconCamera size="s" margin="0 6px 0 0" />
+          <span>{{ $t("BcfComponents.BcfTopicForm.takeSnapshot") }}</span>
+        </BIMDataButton>
+        <BIMDataButton color="primary" outline radius class="m-t-12">
+          <label for="files" class="flex items-center">
+            <BIMDataIconUnarchive fill color="default" size="s" margin="0 6px 0 0" />
+            <span>{{ $t("BcfComponents.BcfTopicForm.importFile") }}</span>
+          </label>
+          <input
+            :disabled="viewpoints.length >= 4"
+            hidden
+            id="files"
+            type="file"
+            multiple
+            accept="image/png, image/jpeg"
+            @change="uploadViewpoints"
```

Comment:
I think you forgot to move that `uploadViewpoints` method here because its still live in `<BcfTopicSnapshotsActions />` component: https://github.com/bimdata/bcf-components/pull/30/files#diff-c862f8f815552e89d5cc4f015b26625af3acce35e41fdbc40efc000574bb683eL94